src/Raven.Client/Documents/DocumentStoreBase.cs

```diff
@@ -42,6 +43,8 @@ protected DocumentStoreBase()
         /// </summary>
         public bool WasDisposed { get; protected set; }
 
+        public abstract DatabaseAiAgents AiAgents { get; }
```

Comment:
and this `AiAgents` -> `AI`, like is requested in the issue.

Reply:
Done